### ISSUE TITLE
./koch runCI should set a custom NIMBLE_DIR to avoid interference; and should honor --nim

### ISSUE THREAD

**Issue by timotheecour** [timotheecour/Nim#755]
XDG_CONFIG_HOME= ./koch --nim:/pathto/nim runCI
runs some commands that sometimes ignore --nim flag eg:

nimble install -y --nimbleDir:build/deps zip

instead:
* NIMBLE_DIR should be set (maybe to a tmp dir)
* --nim should be honored, eg via https://github.com/nim-lang/RFCs/issues/336

## scratch below
NIMBLE_DIR=$HOME/.nimble_fake60 XDG_CONFIG_HOME= nimble install --nim:$nimb -y --nimbleDir:build/deps zip

**Comment by juancarlospaco** [comment]
I wonder if would be better adding support for "dotenv" file format to stdlib ?. 🤔
Would be like 300 lines at most and provide the "best practice" way to handle all those env vars.

**Comment by timotheecour** [comment]
yes, a module to parse dotenv file format (std/formats/dotenv ?) would be useful (refs https://hexdocs.pm/dotenvy/dotenv-file-format.html) and easy to implement (i've implemented this myself as well as the reverse direction from /pathto/bar/baz/foo.txt to $MYENVVAR/fbaz/oo.txt which is also very useful eg for generating less verbose log entries etc)

that said, setting NIMBLE_DIR for runCI can be done independently of that:)
